desktop/font.h -> desktop/gui_layout.h
http://git.netsurf-browser.org/netsurf.git/commit/?id=974a4a21e16a2da85a66aae9b80eeca15ca26dd6

desktop/gui_layout.h -> include/netsurf/layout.h
http://git.netsurf-browser.org/netsurf.git/commit/?id=9ce71876ce500e242e2384c21abb38a8f431138b

desktop/gui_misc.h -> include/netsurf/misc.h
http://git.netsurf-browser.org/netsurf.git/commit/?id=fe7eb85614244d5e0659c88431fbd9726fa33051

--- a/desktop/font_haru.c
+++ b/desktop/font_haru.c
@@ -38,7 +38,7 @@
 
 #include "utils/nsoption.h"
 #include "desktop/font_haru.h"
-#include "desktop/font.h"
+#include "netsurf/layout.h"
 #include "utils/log.h"
 
 
--- a/desktop/font_haru.h
+++ b/desktop/font_haru.h
@@ -27,7 +27,7 @@
 #include <hpdf.h>
 
 #include "desktop/plot_style.h"
-#include "desktop/font.h"
+#include "netsurf/layout.h"
 
 bool haru_nsfont_apply_style(const plot_font_style_t *fstyle,
 			      	HPDF_Doc doc, HPDF_Page page,
--- a/desktop/save_pdf.c
+++ b/desktop/save_pdf.c
@@ -60,7 +60,7 @@
 #include "utils/useragent.h"
 #include "content/hlcache.h"
 #include "utils/nsoption.h"
-#include "desktop/gui_misc.h"
+#include "netsurf/misc.h"
 #include "desktop/gui_internal.h"
 #include "netsurf/bitmap.h"
 
